server/views/components/article-body/index.njk

```diff
@@ -25,6 +25,8 @@
         {% component 'tweet', { tweet: bodyPart.value } %}
       {% elif bodyPart.type === 'instagramEmbed' %}
         {% component 'instagram-embed', { instagramEmbed: bodyPart.value } %}
+      {% elif bodyPart.type === 'blockquote' %}
+        {% component 'blockquote', { body: bodyPart.value.body | safe, footer: bodyPart.value.cite | safe, modifiers: ['small'] } %}
```

Comment:
At some stage we need to fix cardigan to say this is the blockquote, and the other is the pull quote.